docs/source/fitting/fitfunctions/Polynomial.rst

```diff
@@ -0,0 +1,37 @@
+.. _func-Polynomial:
+
+==========
+Polynomial
+==========
+
+.. index:: Polynomial
+
+Description
+-----------
+
+A polynomial function of degree :math:`n` is defined as:
+
+.. math:: y = A_0 + A_1 \times x + A_2 \times x^2 + ... + A_n \times x^n,
+
+where :math:`A_0, ...,  A_n` are constant coefficients.
+
+Usage
+-----
+
+**Example - Fit a polynomial background excluding some peaks
+
+.. testcode:: Ex
+
+    x = np.linspace(-10, 10, 100)
+    y = np.exp(-4*(x+3)**2) + np.exp(-4*(x-3)**2) + 0.1 - 0.001*x**2
```

Comment:
This looks suspect as
`-4*(x+3)**2` + `-4*(x+3)**2` ≡ 2(`-4*(x+3)**2`)
Are these both meant to be the same order `n` here?

Reply:
I think it might be even clearer if I just change the example a bit to only include on peak